ccdproc/tests/test_ccdproc.py

```diff
@@ -619,8 +618,13 @@ def test_catch_transform_wcs_warning():
     def tran(arr):
         return 10 * arr
 
-    with catch_warnings() as w:
-        tran = transform_image(ccd_data, tran)
+    # No warning.
+    transform_image(ccd_data, tran)
+
+    # FIXME: Needs to issue warning when data has WCS.
```

Comment:
How to fix this, @mwcraig ?

Reply:
I think something like this:

```python
ccddata.wcs = WCS()
with pytest.warns():
    _ = transform_image(ccd_data, tran)
```

Reply:
Tried that too. No warning.

Reply:
Ah, that is because the warning is done via `logging.warning` not `warnings.warn` at https://github.com/astropy/ccdproc/blob/main/ccdproc/core.py#L878

Looks like fixing the broken test found a bug. Tested locally with `warnings.warn` and it seems to work as you originally had it.

Reply:
I pushed a fix. Hopefully it is the fix you want! 😸

Reply:
Looks great, thanks again!